docs/prod-install.md

```diff
@@ -60,7 +60,17 @@ fail.  Also note that it automatically logs a transcript to
 `/var/log/zulip/install.log`; please include a copy of that file in
 any bug reports.
 
-## Step 3: Configure Zulip
+## Step 4: Set up email on your host
```

Comment:
Hmm -- how about expanding Step 1 (SSL certs) into a heading like "Set up prereqs: DNS, SSL, and email"? Those are three things you may need to do, or may have basically lying around; and at least two of them (SSL and email) you can do in any order. So it seems reasonable to group them as one "step" with three parts, all optional.